application/src/main/java/com/softserve/lv460/application/entity/SupportedDevice.java

```diff
@@ -0,0 +1,36 @@
+package com.softserve.lv460.application.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+
+@Entity
+@Table(name = "device_template")
+public class SupportedDevice {
```

Comment:
why table is called device_template and entity - SupportedDevice? rename class to DeviceTemplate